asset-managment/niapm.yml

```diff
@@ -0,0 +1,1405 @@
+    type: object
+    properties:
+      key:
+        description: Date representing the beginning of a timespan.
+        type: string
+        example: 2018/11/01
+      value:
+        description: The percentage of time in which the asset was available in a system during the specific timespan.
+        type: number
+        format: double
+        example: 7.38958
+    required:
+      - key
+      - value
+  OverallAvailabilityInTimespanModel:
+    title: Overall Availability In Timespan
+    description: Contains availability information for an asset in a single system for the specified timespan.
+    type: object
+    properties:
+      key:
+        description: The name of the system in which the asset was available.
+        type: string
+        example: APM-PXI1
+      value:
+        description: The percentage of time in which the asset was available in a system during a specific timespan.
+        type: number
+        format: double
+        example: 7.38958
+    required:
+      - key
+      - value
+  ResponseFormat:
+    title: Response Format
+    description: Specifies the return type. Valid options are "JSON" and "CSV".
+    type: string
+    default: JSON
+    enum:
+      - JSON
+      - CSV
+  Destination:
+    title: Request Destination
+    description:
+      Specifies the destination of the request.
+      "INLINE" (default) returns the list of assets as the body of the response.
+      "DOWNLOAD" returns the list of assets as the body of the response and indicates to the client that it should be downloaded as a file.
+      "FILE_SERVICE" sends the list of assets to the file ingestion service and returns the ID of the file to the client in a JSON object.
+    type: string
+    default: INLINE
+    enum:
+      - INLINE
+      - DOWNLOAD
+      - FILE_SERVICE
+  ServicePolicy:
+    title: Service policy
+    description: An object containing the service policy.
+    type: object
+    required:
+      - calibrationPolicy
+    properties:
+      calibrationPolicy:
+        description: Calibration policy
+        type: object
+        properties:
+          daysForApproachingCalibrationDueDate:
+            description: The number of days for an asset to be considered as approaching calibration.
+            type: integer
+            example: 30
+parameters:
+  AssetAvailabilityGranularity:
+    in: query
+    name: granularity
+    description: Specifies the possible granularities that can be used to obtain asset availability.
+    type: string
+    default: WEEK
+    enum:
+      - NONE
+      - WEEK
+      - MONTH
+  AssetId: 
+    in: path
+    name: assetId
+    description: Specifies the identifier of an asset.
+    type: string
+    required: true
+    x-example: "29162;01B245D6;4243;0"
+  Skip:
+    in: path
+    name: skip
+    description:
+      How many resources to skip in the result when paging.
```

Comment:
If we were to phrase this description as "The number of resources to skip..." would this describe the same action here? Or is it technically inaccurate? If it's OK, then I suggest we replace all of the "How many" phrases with "The number of" instead.